applications/MeshingApplication/CMakeLists.txt

```diff
@@ -129,6 +138,87 @@ else(${INCLUDE_MMG} MATCHES ON)
     target_link_libraries(KratosMeshingCore PUBLIC KratosCore tetMeshOpt)
 endif(${INCLUDE_MMG} MATCHES ON)
 
+target_link_libraries(KratosMeshingCore PUBLIC KratosCore triangle tetMeshOpt)
+set_target_properties(KratosMeshingCore PROPERTIES COMPILE_DEFINITIONS "MESHING_APPLICATION=EXPORT,API")
+
+IF(NOT DEFINED PMMG_LIBRARY_PREFIX)
+    SET(PMMG_LIBRARY_PREFIX "lib")
+ENDIF(NOT DEFINED PMMG_LIBRARY_PREFIX)
+
+if(${INCLUDE_PMMG} MATCHES ON)
+    IF(NOT DEFINED INCLUDE_MMG)
+        message( FATAL_ERROR "MMG not included. Please add -DINCLUDE_MMG=ON to the configure file in order to compile with ParMMG")
+    ELSE(NOT DEFINED INCLUDE_MMG)
+        if(${INCLUDE_MMG} MATCHES OFF)
+            message( FATAL_ERROR " MMG disabled. Please set -DINCLUDE_MMG=ON in the configure file in order to compile with ParMMG")
+        endif(${INCLUDE_MMG} MATCHES OFF)
+    ENDIF(NOT DEFINED INCLUDE_MMG)
+
+    add_definitions( -DINCLUDE_PMMG )
+    ## find Metis for ParMmg
+    if(NOT DEFINED USE_METIS_5)
+        message("USE_METIS_5 not defined. Setting to ON")
+        SET (USE_METIS_5 ON)
+    endif(NOT DEFINED USE_METIS_5)
+
+    if(${USE_METIS_5} MATCHES ON)
+        message("using metis (and not Parmetis) version 5")
+        add_definitions(-DKRATOS_USE_METIS_5)
+        find_package(METIS REQUIRED)
+        include_directories(${METIS_INCLUDE_DIR})
+        link_directories(${METIS_LIBRARIES})
+        install(FILES ${METIS_LIBRARIES} DESTINATION libs)
+    else(${USE_METIS_5} MATCHES ON)
+        message("using PARMETIS < 4 as default. Set USE_METIS_5=ON to compile with METIS v5 instead- note that current compile option is deprecated and will be removed soon")
+        find_package(PARMETIS REQUIRED)
+        include_directories(${PARMETIS_INCLUDE_DIR})
+        link_directories(${PARMETIS_LIBRARIES})
+    endif(${USE_METIS_5} MATCHES ON)
```

Comment:
please remove this (I know is the wrong PR to comment but otherwise i forget again)

We recently discussed to drop the support for metis < 5 (in the MetisApp), hence you can remove it from here

Reply:
Ok noted. As you say this will be on #7790